### DISCUSSION TITLE
Allow Customer Creation without Email

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#388] [Ideas]
Hi Team,

In India, most people do a phone orders and we want to store such customer details too. In such situation, we don't have an option to enter customer email. So it will be better if you provide support for adding customer without email address.

**Comment by Blair2004** [comment]
What about entering a fake email ? For example, 123456@storeemail.com

**Reply by rsidhaarth** [reply]
I don't think that is a proper solution as the store will still try to send emails to that fake email and email logs will be filled and transactional emails will be used. It will be a better option if you make email not mandatory.
